app/src/routes/home/index.js

```diff
@@ -8,6 +8,6 @@ const ctrl = require("./home.ctrl");
 const webtoonctrl = require("../webtoon/webtoon.ctrl");
 
 router.get("/", ctrl.output.home);
-router.get("/api/webtoon", webtoonctrl.process.getWebtoon);
+router.get("/api/webtoon", webtoonctrl.process.getWebtoons);
```

Comment:
엔드포인트는 일반적으로 복수형을 사용하는 경우가 많습니다. 
웹툰 목록 전체를 조회 => `/api/webtoons`처럼 복수형 자원명을 사용하는 게 의미 전달 측면에서도 정확합니다.

사실 RESTful API에 엄격한 규칙은 없지만, 대부분의 설계 가이드라인과 실제 프로젝트에서 자원명에 복수형을   사용하는  경우가 많기 때문에 복수형 사용을 추천드립니다.
```suggestion
router.get("/api/webtoons", webtoonctrl.process.getWebtoons);
```

Reply:
변수명에는 전체 조회라는 의미로 s를 붙였었는데, 엔드포인트도 같은 맥락으로 복수형을 사용하는게  전달성도 좋고 일관성 있네요!